docker-compose.cdr.yaml

```diff
@@ -151,9 +151,9 @@ services:
       - TENANT=${TENANT}
       - GIN_MODE=debug # change to release to disable gin debug
       - SPEC=cdr
-      - BANK_URL=http
       - OPENBANKING_SERVER_ID=cdr
       - BANK_URL=http://bank:8070
+      - ENABLE_TLS_SERVER=true
```

Comment:
I would actually like to keep this option consistent with what was done for the consent-page. The configuration variable name is the same, and the default is set to true in case it wasn't explicitly supplied.
`EnableTLSServer  bool          `env:"ENABLE_TLS_SERVER" envDefault:"true"``

But I can probably go ahead and change it in the consent page too.